nx.json

```diff
@@ -59,6 +63,9 @@
     "generate": {
       "dependsOn": ["^build"]
     },
+    "generate:addresses:check": {
+      "dependson": ["deployments", "^deployments", "scripts"]
+    },
```

Comment:
This config makes nx smart enough to know that the address generation script check only ever needs to run when the artifacts actually have changed. Otherwise cache test result